bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/RuleEngine.java

```diff
@@ -1015,4 +1023,14 @@ public void templateUpdated(Collection<Template> templates) {
 
     }
 
+    public void setEventPublisher(EventPublisher ep) {
+        this.eventPublisher = ep;
```

Comment:
Why? I think it was introduced especially for the RuelStatusInfoEvents, so that we don´t need the EventPublisher within the RuleEngine.